MODULE.bazel

```diff
@@ -0,0 +1,32 @@
+module(name = "nanopb")
+
+bazel_dep(name = "bazel_skylib", version = "1.5.0")
+bazel_dep(name = "rules_cc", version = "0.0.9")
+bazel_dep(name = "rules_python", version = "0.31.0")
+bazel_dep(name = "rules_proto", version = "6.0.0-rc2")
```

Comment:
Is the -rc2 version because the current release doesn't support bazel modules yet? I'd suggest a comment or holding off on merging the PR until released

Reply:
no specific reason, just picked whatever latest https://registry.bazel.build/modules/rules_proto.
there is an older version `5.3.0-21.7` from bcr, too, which I think it implies bzlmod support is up, rc2 might refer to some new features of protobuf repo. but anyways I am not quite familiar with the naming convention for that repo, if you have a preferred version, I can modify it.

Reply:
I would use the latest released versions that are not `-rc` (release candidate) or `-alpha` versions, please update those and confirm they still work

Reply:
there is no stable version for `rules_proto_grpc`, I think the decision has to be made to use `http_archive` or `alpha`, any thoughts?